### PR TITLE
lints and release

### PR DESCRIPTION
**Pull request by kevmoo**

### CODE REVIEW

File: mono_repo/CHANGELOG.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
-* Add `recursive` global flag. When set, we will walk all sub-directories looking for `pubspec.yaml` files.
-
 ## 0.2.0
```
Comment (alorenzen) [inline mono_repo/CHANGELOG.md]: Looks like I forgot the CHANGELOG in #38 ...

Comment (kevmoo) [reply mono_repo/CHANGELOG.md]: Want to throw me some text to add in?

Comment (alorenzen) [reply mono_repo/CHANGELOG.md]: * Add `before_script` support to the `travis` command. When that value is set in a `travis.yml` file, we will call the script before running any of the tasks for that package.

**Comment by kevmoo** [comment]
@alorenzen – published as v0.2.0

**Pull request merged**